templates/zerver/help/invite-a-friend-to-zulip.md

```diff
@@ -10,32 +10,25 @@ their Zulip organization. To do so, simply follow the following steps.
 
     ![settings](/static/images/help/invite-dropdown.png)
 
-3. After clicking **Invite users**, a modal window titled **Invite
-   users to Zulip** will appear.
+3. After clicking **Invite users**, a modal window titled
```

Comment:
Shouldn't step be something done by a user? I think this can be merged with either second or fourth step.